trunk/src/lsc/lscabsphotdef.py

```diff
@@ -300,6 +300,9 @@ def absphot(img,_field='',_catalogue='',_fix=True,rejection=2.,_interactive=Fals
     elif 'fa' in _instrume:
         colorefisso = {'uug': 0.0, 'ggr': 0.109, 'rri': 0.027, 'iri': 0.036, 'BBV': -0.024, 'VBV': -0.014,
                        'UUB': 0.059, 'BUB': -0.095, 'VVR': -0.059, 'RVR': -0.028, 'RRI': -0.033, 'IRI': 0.013, 'ziz': -0.04}
+    elif 'mc' in _instrume:
```

Comment:
are you sure _instrume contains 'mc'? When I look at the headers for GRB221014 in the INSTRUME header keyword (and print it here) it is epXX (e.g. ep05)